main.go

```diff
@@ -62,6 +62,7 @@ func main() {
 		" maintained. If does not exists, bingo list will fail.")
 	// Go flags is so broken, need to add shadow -v flag to make those work in both before and after `list` command.
 	listVerbose := listFlags.Bool("v", false, "Print more'")
+	listJSON := listFlags.Bool("json", false, "output to json")
```

Comment:
What about `-o / --output option` that would accept empty string (default tab view) and "json" value? This will be format compatible with future formats e.g yaml (: